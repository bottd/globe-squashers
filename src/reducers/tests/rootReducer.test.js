import {rootReducer} from '../';

describe('rootReducer', () => {
  it('Should match snapshot', () => {
    expect(rootReducer).toMatchSnapshot();
  });
});
