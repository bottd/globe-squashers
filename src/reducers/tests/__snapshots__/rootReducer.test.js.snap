// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rootReducer Should match snapshot 1`] = `[Function]`;
